### DISCUSSION TITLE
Have you built this adapter?

### DISCUSSION THREAD

**Discussion by Franticware** [Franticware/usb-to-playstation-mouse#1] [General]
Here is the place to share:
* your impressions, how it works for you
* what games you use it for
* which mouse model/type/brand works for you (and which does not)
* photos and videos of your build and gameplay

**Comment by jankfactor** [comment]
@Franticware thank you so much for this, it works perfectly! :) I'd been looking for a PS1 mouse on eBay but they were silly money. I'm glad I came across this project.

So far I've been testing it with `Quake II`, `Syndicate Wars` and `Warzone 2100` with great success.

[Here's a little footage of Syndicate Wars with PS1 mouse support. ](https://www.youtube.com/watch?v=Lj-xOf_CR6o)

Syndicate Wars and Warzone 2100 both seem to need the mouse in Port 1 and controller in Port 2 (whereas Quake II is the other way around). Syndicate in particular I'm very pleased with, as it's much easier to control and much closer to the DOS original now.
![PS1-Mouse](https://user-images.githubusercontent.com/18681942/182867370-9c5b2a48-db72-4daa-b708-e2d427afc786.jpeg)

**Comment by psyrax** [comment]
Decided that a mouse and a memory card in the same controller port would be a great idea. And it worked, Using a regular rpi pico for the mouse and a nano for the memory card. 

![42770F60-69B7-4567-B22E-3A7AF1BBD896](https://user-images.githubusercontent.com/1424135/187742024-7ebe4a89-520a-45a4-a50f-7713cb26a094.jpeg)
![672A0B0D-4DD7-409F-B6A4-515E284FCAEB](https://user-images.githubusercontent.com/1424135/187742040-66c077f6-e18f-4f2f-a790-11894a292542.jpeg)

**Comment by riadabdel** [comment]
Hello,
Thanks for the tuto, its working with my mouse!
Now, i'd like to play with my pxn 0082 fight stick but it's not working. What is the best way to recognise my pico as a gamepad ?

**Reply by Franticware** [reply]
Hi, glad to hear positive feedback. As for the gamepad, the PlayStation controller port protocol supports various devices and has distinct format for each type. This project currently only implements mouse protocol, and the USB host also detects just mouse. But it would certainly be possible to add gamepad detection and readout to the USB host, which could then be translated into PlayStation protocol and output to the controller port. Pico has the advantage of having good documentation and enough sample code, so it should be relatively straightforward. Having a logic analyzer (like Saleae) is also very handy so that you can see what data is actually going back and forth through the controller port.

**Comment by gwald** [comment]
![Time Crisis](https://cdn.discordapp.com/attachments/827671112480718863/1178177447590637628/IMG20231126143251.jpg?ex=65753240&is=6562bd40&hm=d659db28601f74bf27f61b91365c6a0e118b797f0503e93f48ababe542da9894&)

(The adapter is infront of the TV, it's wrapped in black tape so it's hard to see)

Thank you for this project!

I just finished it... It worked well on time crisis.

The mouse pointer is blue and in the middle, but there's no mouse pointer in the actual game! :( which makes it impossible to actually play it, having to remember the mouse location on screen.

The mouse is just a generic dell mouse.

some feedback:

It's a pretty simple project for beginners, the hardest part for me was soldering the two points behind the micro USB port, I needed a magnifying holder. If these points are somewhere else on the board that's better to solder to, that would make it completely noob friendly.

Also, I would recommend updating the wiring image, the white wires are very hard to see, so maybe outline them or use a different colour.


Thanks again!
:D

**Reply by Franticware** [reply]
Hi and thank you for the feedback. I have a question concerning the wiring image. Could you provide the screenshot and browser information? In Firefox/Linux, I do see the outline fine.

**Reply by gwald** [reply]
ha, it's SVG... I was using my mobile when soldering it up, my default mobile browser is brave. It converted all the black lines into white :/
I didnt think to double check on PC... anyway good thing it still works :D

**Comment by RangerKoc** [comment]
https://www.youtube.com/watch?v=HElEyNB2lAg

**Comment by gentleben** [comment]
Thanks a lot for this project. It works a treat!
![PS1USB2](https://github.com/Franticware/usb-to-playstation-mouse/assets/638249/001ffca5-fdee-4ea5-a83b-8f043aa40e8c)
![PS1USB1](https://github.com/Franticware/usb-to-playstation-mouse/assets/638249/f607d599-5f21-46de-8cd8-e2577b413a0c)

**Comment by Mait-0** [comment]
Thanks a lot!

I am able to hotswap between mouses. Anyone?

Works for:
Logitech M100
Logitech M-UV96

Do NOT works for:
Logitech Unifiying M235 2gen

![IMG20240410180029](https://github.com/Franticware/usb-to-playstation-mouse/assets/166638671/f17bf631-1456-41af-9bd0-e954f44efb2c)

**Comment by Franticware** [comment]
![IMG_20240602_191702656~2](https://github.com/Franticware/usb-to-playstation-mouse/assets/71665922/c4c2dd36-f1d2-4209-8b66-bf95f51ef470)

Working on an overhaul, here is my prototype. The project will be continued in a new repository: https://github.com/Franticware/usb-to-ps1-mouse-pro

**Reply by hkcisonline** [reply]
Made by a Friend for m'y total pleasure 😁![edited_IMG_20240611_133613.jpg](https://github.com/Franticware/usb-to-playstation-mouse/assets/67123297/d3181477-fd39-417f-8318-91a5ea23e1fd)

**Comment by MukiEX** [comment]
I built a pair of them using the new "Pro" schematic" and with a 2nd RP2040 on each (running HID-Remapper) I got an Egret II Mini working as a mouse for the PlayStation version of Puchi Carat.

![dEsZWGW](https://github.com/user-attachments/assets/89b59318-a711-4392-8e09-045a9d9cf1b1)

Closeup of the adapters:

![XpUyETX](https://github.com/user-attachments/assets/18c19462-30a2-41c3-9805-3d9306d76028)

I'm messing with a slightly different PCB revision to use a PSX cable with an RJ45 connector on the PS360/Brook Retro pinout. If it works I'll upload a photo and send you a copy of the modified KiCad. (the changes look absolutely terrible)

**Reply by MukiEX** [reply]
It worked! My soldering is a mess, but it was my first PCB and I was excited to get it working. JLCPCB couldn't source the inductor at the time, so I had to order a pack of them on Ali and I initially futz up the soldering job, but once I fixed it, it came up!

I have a couple of RJ45 cables I've terminated myself from PS1/PS2 controllers destined for recycling (the controllers themselves were already destroyed, the cables were going to be scrapped as well). Two of them are about a foot long (below) and work perfectly. One is much longer and works as long as I don't daisy-chain a 2nd RP2040 (with HID-Remapper installed).

But it works! I attached my modified KiCad_PCB file, also below. For what it's worth, I used Filthy Pants's PS360 pinout guide to terminate my controller cables:
http://filthypants.blogspot.com/2018/12/retro-console-rj45-pinouts-ps360-mc.html

[usb2ps1mouse_rj45.zip](https://github.com/user-attachments/files/17171535/usb2ps1mouse_rj45.zip)


![PSX_USB_RJ45](https://github.com/user-attachments/assets/2a9af419-4569-4aa2-a5c4-360fb71670ce)

**Reply by Franticware** [reply]
This is cool. Do I understand correctly that JLCPCB soldered some components for you?

**Reply by MukiEX** [reply]
Indeed! I asked them to do all the tiny surface mount stuff. It probably cost about as much as it did to make as the PCBs themselves though.

The only things I soldered myself were:

- The RP2040-Zero
- The RJ45 Jack
- The USB-A Jack
- The 220 inductor

BTW, thank you so much for making this project. Using the Egret II Mini on a real PlayStation for Puchi Carat's been fun ^o^

**Reply by MukiEX** [reply]
I'm messing with merging the Core1 (PSX) code from this and the Core0 (it only runs on 1 thread) code from HID-Remapper. I'm not doing a great job of it, but hopefully eventually I can plug this straight into an Egret II Mini (or even a Switch controller) and just extract the device type/HID reports from the Remapper side to send out retro-style to the controller side. ^o^

**Comment by lucentibarretos-source** [comment]
![17706483123192144035330225771647](https://github.com/user-attachments/assets/70cb9dcb-ea0c-4b84-ac4a-cbbbdcde016d)
![17706481721746373474461974051066](https://github.com/user-attachments/assets/5c9aeb38-23de-4bb5-90e6-3d8c6a3235b3)
![17706481948213804753998994679225](https://github.com/user-attachments/assets/f5734384-810f-4687-8af3-ca3d0003063e)
 Fiz vários testes ficou muito bom

**Comment by brennier** [comment]
Thank you so much for this project! I recently built my own adapter using your original firmware and wiring schematic. 

Unfortunately, most of my mice had compatibility problems, but I can confirm that my Logi M185 2.4 GHz wireless mouse works perfectly!

I can also confirm that my Japanese SCPH-5500 console does NOT support using the mouse in the system menu.

<img width="3000" height="4000" alt="72e4adfe-7eba-49b2-8522-4d829e965472" src="https://github.com/user-attachments/assets/51a08c3e-66c0-4f28-9cb4-41ef0200c5cc" />
<img width="3000" height="4000" alt="7823c80b-da31-4240-98b2-4f95c96490d1" src="https://github.com/user-attachments/assets/c0e36acf-009b-41a5-99dc-f1dcac3bb0f2" />

**Comment by lucentibarretos-source** [comment]
Olá bom Dia, maravilha, meus mouse sem fio não tive sorte não tiveram
compatibilidade, já os mouse com fio todos que eu testei funcionaram

Em sáb., 15 de ago. de 2026, 06:59, 上江洲 ベネット (Bennett Rennier) <
***@***.***> escreveu:
